posts/coalesce2022-a-recap.md

```diff
@@ -0,0 +1,238 @@
+
+![Snowplow User Stitching](https://raw.githubusercontent.com/ippontech/blog-usa/master/images/2022/11/coalesce_snowplow_user_stitching.png)
+
+Snowplow then runs a query using the mapping to generate a table containing the domain_userid, the most recent user_id, and the processed date. This table allows Snowplow to see a person's identifier before they even log in for the day. To scale this, Snowplow has a custom incremental materialization called snowplow_incremental, reducing the amount of data scanned. All of this is powered by dbt!
+
+[Watch the full talk here!](https://www.youtube.com/watch?v=uCHtOLsCsAA&t=1443s)
+
+## Beyond the buzz: 20 real metadata use cases in 20 minutes with Atlan and dbt Labs
+
+In this talk, Prukalpa Sankar, Co-founder of Atlan presents 20 scenarios in which data teams have used metadata to drive their work as well as how metadata and the dbt Semantic Layer can transform how a team works with metrics. 
+
+The chief question that Prukalpa and Atlan is trying to address is, `When I'm looking at data in a dashboard and am wondering if it is correct, is it possible to get the answers immediately?` 
+
+Let's get right into the 20 real-life use cases mentioned in the title:
+1. Streamline analyst 'service' requests
+> Problem: Constant questions and asks for analysts from business users. 
+> - What does metric X mean? How is it different from Metric Y? 
+> - Can someone confirm if this is the right way to join these tables? Here's the SQL statement
+> - I want to see MEtric X by geography instead of segment?
+> 
+> Solution: With metadata, you can actually go from requests to products. You can consider each of these requests as internal reusable products for their organization. Data, Metrics and Queries can all be products for your organization.
+
+2. Streamline Data/Analytics Engineer 'service' requests
+> Problem: Constant questions and asks for engineers from other data users
+> - Hey, is this data updated?
+> - Do we have this data? 
+> - Do we ingest data from our survey tool into the warehouse?
+> - Is there an existing Tableau extract for ROI by new/existing inventory?
+> - How are columns for this table derived?
+>
+> Solution: You can extend data-as-a-product one step further. You can bring in context that a data engineer needs, such as Airflow SLAs (auto-updated), freshness from dbt tags, autoamted column level lineages, etc.
+
+3. Speed data team onboarding and ramp up
+> Problem: How do you speed up team onboarding and ramp-up?
+> Typically analysts spend 18 months on the job, and it takes them 6 months to onboard into an organization. 
+> 
+> Solution: By making assets (such as your data objects, metrics and queries) you can reduce this time astronomically.
+
+4. Speed up root-cause analysis
+> Problem: Finding the solution to `It doesn't seem like Metric X is right?` is far from easy
```

Comment:
Some "Problem" lines have punctuation some don't.  It's okay to leave off I suppose (like you do in the numbered headers), just make sure to do it consistently across the blog post.